bot/resources/easter/save_the_planet.json

```diff
@@ -0,0 +1,71 @@
+[
+    {
+        "title": "Save the Planet: Choose Reneweable Energy",
+        "image": {"url": "https://cdn.dnaindia.com/sites/default/files/styles/full/public/2019/07/23/851602-renewable-energy-istock-072419.jpg"},
+        "footer": {"text": "The best thing you can do is sharing this information!"},
+        "fields": [
+            {
+                "name": "The Problem",
+                "value": "Getting energy from oil or fossil fuels isn't a good idea, because there is only so much of it.",
+                "inline": false
+            },
+
+            {
+                "name": "What you can do",
+                "value": "Use renewable energy, such as wind, solar, and hydro, because it is healthier and is not a finite resource!",
+                "inline": false
+            }
+        ]
+    },
+
+    {
+        "title": "Save the Planet: Save the Trees!",
```

Comment:
In SeasonalBot embed titles, we generally only capitalise the first letter (and names, cities and similar, of course).
- `Save the Trees` should be `Save the trees`
- `Choose Reneweable Energy` should be `Choose reneweable energy`

Reply:
Ok